src/receive_imf.rs

```diff
@@ -245,12 +245,16 @@ pub(crate) async fn receive_imf_inner(
     }
 
     if let Some(ref status_update) = mime_parser.webxdc_status_update {
-        if let Err(err) = context
-            .receive_status_update(from_id, insert_msg_id, status_update)
-            .await
-        {
-            warn!(context, "receive_imf cannot update status: {}", err);
-        }
+        // protect against users which falsely sent status updates because they
+        // didn't yet receive the group-leave message
+        if is_contact_in_chat(context, chat_id, from_id).await? {
+            if let Err(err) = context
+                .receive_status_update(from_id, insert_msg_id, status_update)
+                .await
+            {
+                warn!(context, "receive_imf cannot update status: {}", err);
+            }
+        };
```

Comment:
maybe add an else branch and print a warning for debugging

Reply:
unresolving as the `else` was removed, probably accidentally, by one of the last commits.

Reply:
resolving again as this does not work anyway as expected :) see https://github.com/deltachat/deltachat-core-rust/pull/3568#pullrequestreview-1102024048